CMakeLists.txt

```diff
@@ -21,43 +21,73 @@ if(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.12")
     cmake_policy(SET CMP0074 NEW)
 endif()
 
-find_package(ZLIB REQUIRED)
-include_directories(${ZLIB_INCLUDE_DIRS})
-
-set(spng_SOURCES spng/spng.c)
-
 if(NOT CMAKE_HOST_WIN32)
-    set(spng_LIBS -lm ${ZLIB_LIBRARIES})
+    set(MATH_LIBRARY "-lm")
```

Comment:
This should probably be just `m`, as that'll get translated to the appropriate flag by CMake